Fixing main.m

hkirvesl opened the pull request:
This fixes the main.m, which was missing type parameter when calling globalize.

hkirvesl committed:
Add files via upload

fixed main.m

hkirvesl committed:
Add files via upload

fixed main.m

hkirvesl commented:
This is the pull-request for enabling the local, sequential run. The gorgon merge is a separate pull-request

JulieWinchester reviewed (changes requested):
Is there a reason why there is a code/main.m file and a separate main.m file? Are they identical? In general, all code files should be in the code directory. 

Also, I'm looking at lines 73 and 130, is there a reason for using a space between function arguments in the first and not in the second? 

Thanks!

hkirvesl committed:
delete main.m from the main directory

The main.m was accidentally copied to two locations, and it is indeed not needed in the main directory. The correct file is /code/main.m

hkirvesl committed:
Fixing the non-existent jobs problem for cluster

Bug-Fix: Non-existing jobs for cluster MST. Formatting the output directory so that the cluster MST can read the files from a correct folder.

hkirvesl committed:
Undoing the previous commit

Mistake in the previous commit: The cluster bug was in the merged version. Reverting the change manually.

hkirvesl commented:
Merging as per discussion on Nov 2 meeting
